### PR TITLE
Tests fixes documentation

### PR DESCRIPTION
**Pull request by hnargos**
added 3 small tests for the potential
fixed a couple of doc strings
everyone should crtl + f search for COMMENTS or their specific github names some have a comment directed to them

### CODE REVIEW

**Comment by hnargos** [comment]
I am not entirely happy how that looks in the test_api.py file but it should be good enough for now.

**Comment by harmening** [comment]
The file api_test.py is failing, because the command get_ipython().magic(u'matplotlib inline') in line 16 can only be run with ipython and not with python. I don't really understand: What do we the need the api_test.py for? Our tests are located in test_api.py...

**Review by harmening — COMMENTED** [review]
If you like you could constrain every line length to have a maximum of 79 characters for reasons of clarity and comprehensibility. But the rest looks fine! I really like your symmetry tests for the potential :-)

**Comment by hnargos** [comment]
Every comment line or generally every line of code?

**Comment by Dr-Manni** [comment]
looks good! Thanks for fixing the dosctrings
However lets wait before we merge. 
Darian's open PR will change Input Paramters, and the cutoff radii will be computed within MD.py 
I propose we merge Darians PR first, then adjust Initial_Test_Parameters.py accordingly and finally merge this PR. I will adjust the force tests, to match your test system.

**Comment by hnargos** [comment]
Yeah, I just wanted to put that out there so that no double work is done.

**Comment by harmening** [comment]
generally every line

**Comment by skieninger** [comment]
@harmening The api_test is not a part of the software-package yet. We just use it to test all functions, the user can use from package md (get_potential/forces/neighbourlist/energy,  propagate_system etc). We can delete this file at the end or leave it as blueprint.

@hnargos I had a look at your changes. There are some changes, that will turn out bad for the connectivity of the single code parts. Please do not change:

- Initial_Parameters.py line 21. We absolutely need L as an array. Changing this will crash the whole thing.

- Initial_Test_Parameters: Please have a look at the updated Initial_Parameters and add changes to your file. 

- particle_interaction.py: Please don't delete line 5 as we need erf. It's more efficient to reserve memory for a matrix beforehand as Darian does it in his code. Please don't delete lines 157-159. Moreover line 558 needs to stay as well. Marco will explain that on Tuesday after lecture.

Thank you for updating the docstrings and the test functions. The symmetry test is really nice.

**Comment by skieninger** [comment]
@harmening I forgot sth: We run the tests in api_test.ipynb and the api_test.py gets generated automatically. We don't need api_test.py at all, so we can just delete it and problems are solved.

**Comment by harmening** [comment]
@skieninger: Nice, so we can gitignore the api_test.py and delete it from the repo :-)

**Comment by skieninger** [comment]
@hnargos Sorry, the erf-function doesn't get used, you're right. My fault, you can remove it.

File: md/particle_interaction.py [hunk 1]
```diff
@@ -383,7 +417,7 @@ def __long_range_forces(self,Positions,Labels):
 
         Labels: Nx? Array
             Array with N rows and ? Columns. The third Column should contain labels, that specify the chemical species of the Particles.
-            Particle A should have the label 1 and Particle B should have the label 0.
+            Particle A should have the label 1 and Particle B should have the label 0. The first column contains the masses, the second the charge.
```
Comment (Dr-Manni) [inline md/particle_interaction.py]: Its created the other way:
label(A) = 0
label(B) = 1

File: md/particle_interaction.py [hunk 2]
```diff
@@ -281,8 +313,9 @@ def __long_range_energy(self,labels,positions):
 
                 Parameters
                 ---------------
-                charges: N x 1 Vector
-                    Vector with the charge of each particle. The index indicates the particle number.
+                labels: Nx? Array
+                    Array with N rows and ? Columns. The third Column should contain labels, that specify the chemical species of the Particles.
+                    Particle A should have the label 1 and Particle B should have the label 0. The first column contains the masses, the second the charge.
```
Comment (Dr-Manni) [inline md/particle_interaction.py]: See Below, 
label(A) = 0
label(B) = 1

File: md/particle_interaction.py [hunk 3]
```diff
@@ -327,7 +361,7 @@ def __short_range_forces(self,Positions, Labels,L):
 
         Labels: Nx? Array
             Array with N rows and ? Columns. The third Column should contain labels, that specify the chemical species of the Particles.
-            Particle A should have the label 1 and Particle B should have the label 0.
+            Particle A should have the label 1 and Particle B should have the label 0. The first column contains the masses, the second the charge.
```
Comment (Dr-Manni) [inline md/particle_interaction.py]: See Below, 
label(A) = 0
label(B) = 1

File: md/particle_interaction.py [hunk 4]
```diff
@@ -519,14 +563,16 @@ def compute_forces(self,Positions,Sigma, Epsilon, Labels,L, switch_parameter, r_
 
         Labels: Nx? Array
             Array with N rows and ? Columns. The third Column should contain labels, that specify the chemical species of the Particles.
-            Particle A should have the label 0 and Particle B should have the label 1.
+            Particle A should have the label 1 and Particle B should have the label 0. The first column contains the masses, the second the charge.
```
Comment (Dr-Manni) [inline md/particle_interaction.py]: See Below, 
label(A) = 0
label(B) = 1

File: md/particle_interaction.py [hunk 5]
```diff
@@ -555,7 +601,7 @@ def compute_forces(self,Positions,Sigma, Epsilon, Labels,L, switch_parameter, r_
             Positions_Difference[:,2] = Positions_Difference[:,2]%(L[2]/2)
             for j in neighbours[i]:
                 #Calculate the Distances 
-                dist = np.linalg.norm(Positions_Difference[j,:])
+                dist = np.linalg.norm(Positions_Difference[j,:])  #COMMENT (@skieninger): I think you can use the absolut distances we get from the neighbour list. dist == distances[i][j]
```
Comment (Dr-Manni) [inline md/particle_interaction.py:629]: thats My Part, 
I Agree that Dists could be importesd and used AT that Point,  but I need Positions_difference below

**Pull request merged**